classes/Config.js

```diff
@@ -116,22 +116,28 @@ class CollectionConfig extends Config {
     }
   }
 
-  async addItemToOrder(item) {
-    const collectionName = this.collectionName
-    
-    const { content, sha } = await this.read()
+  async read() {
+    const { content, sha } = await super.read()
     const contentObject = yaml.safeLoad(base64.decode(content))
-    
-    let index
-    if (item.split('/').length === 2) {
-      // if file in subfolder, get index of last file in subfolder
-      index = _.findLastIndex(
-        contentObject.collections[collectionName].order, 
-        (f) => f.split('/')[0] === item.split('/')[0]
-      ) + 1
-    } else {
-      // get index of last file in collection
-      index = contentObject.collections[collectionName].order.length
+    return { contentObject, sha }
+  }
+
+  async addItemToOrder(item, index) {
+    const collectionName = this.collectionName
+    const { contentObject, sha } = await this.read()
+
+    if (index === undefined) {
+      let index
```

Comment:
Can we use a different name instead of `index`? Since this clashes with the method argument

Reply:
As discussed offline, we'll leave it as it is since there aren't any errors